apps/src/applab/commands.js

```diff
@@ -808,7 +808,10 @@ applabCommands.setText = function (opts) {
     } else if (element.tagName === 'IMG') {
       element.alt = opts.text;
     } else {
-      element.textContent = opts.text;
+      var lines = opts.text.split('\n');
+      element.innerHTML = lines[0] + lines.slice(1).map(function (line) {
+        return '<div>' + (line.length ? line : '<br>') + '</div>';
+      }).join('');
```

Comment:
This also seems complex enough to deserve a comment. Possibly also a mochaTest? I wouldn't think it would be too difficult.